goenvtemplator.go

```diff
@@ -80,6 +80,10 @@ func main() {
 	flag.BoolVar(&printVersion, "version", false, "Prints version.")
 	var envFileList envFiles
 	flag.Var(&envFileList, "env-file", "Additional file with environment variables. Can be passed multiple times.")
+	var delimLeft string
+	flag.StringVar(&delimLeft, "delim-left", "", "Override default left delimiter `{{`. You should consider overriding also right delimiter.")
+	var delimRight string
+	flag.StringVar(&delimRight, "delim-right", "", "Override default right delimiter `}}`. You should consider overriding also left delimiter.")
```

Comment:
I would omit the part about considering overriding opposite delimiter. User is clever.

Reply:
ok let's hope
fixed in https://github.com/seznam/goenvtemplator/pull/14/commits/6498030ac29eb46547c7b593be3a2fb254d6a284